docs/howtos/raw-policies.md

```diff
@@ -0,0 +1,283 @@
+Raw policies can access context-aware[ capabilities](../explanations/context-aware-policies.md) like standard policies.
+
+## Running a Policy Server inside Kubernetes without the Kubewarden controller
+
+It's not possible to use a Policy Server instance managed by the Kubewarden controller to host raw policies.
+The controller will not allow the user to change the Policy Server ConfigMap to add a raw policy,
+since it will try to reconcile it reverting the changes.
+Because of that, a dedicated Policy Server has to be started.
+
+Create a `policy-server.yaml` file with the following content:
+
+```yaml
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: policy-server-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: policy-server
+  template:
+    metadata:
+      labels:
+        app: policy-server
+    spec:
+      containers:
+        - name: policy-server
+          image: ghcr.io/kubewarden/policy-server:1.9.0
+          ports:
+            - containerPort: 3000
+          volumeMounts:
+            - name: policy-store
+              mountPath: /registry
+            - name: policies-config
+              mountPath: /policies.yml
+              subPath: policies.yml
+      volumes:
+        - name: policy-store
+          emptyDir: {}
+        - name: policies-config
+          configMap:
+            name: policies-configmap
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: policy-server-service
+spec:
+  selector:
+    app: policy-server
+  ports:
+    - protocol: TCP
+      port: 3000
+      targetPort: 3000
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: policies-configmap
+data:
+  policies.yml: |
+    raw-validation:
+      url: ghcr.io/kubewarden/tests/raw-validation-policy:v0.1.0
+      settings: 
+        validUsers:
+          - alice
+          - bob
+        validActions:
+          - read
+          - write
+        validResources:
+          - orders
+          - products
+    raw-mutation:
+      url: ghcr.io/kubewarden/tests/raw-mutation-policy:v0.1.0
+      allowedToMutate: true
+      settings:
+        forbiddenResources:
+          - privateResource
+          - secretResource
+        defaultResource: publicResource
+```
+
+Apply the configuration:
+
+```bash
+kubectl apply -f policy-server.yaml
+```
+
+:::info
+The Policy Server instance deployed will have access to Kubernetes resources that could be leveraged by context aware policies.
+The access level to the Kubernetes resources is determined by the Service Account used to run the Policy Server workload.
+
+In the previous example, no Service Account is defined inside of the Deployment specification; hence the `default` Service Account is going to be used.
+:::
+
+## Using the validate_raw endpoint
+
+### Validation
+
+The raw validation endpoint is exposed at `/validate_raw` and accepts `POST` requests.
+
+Let's try to validate a JSON document against the `raw-validation` policy:
```

Comment:
maybe worth it to provide people with a port-forward:
`kubectl port-forward service/policy-server-service 3000:3000 -n default`